.travis.yml

```diff
@@ -1 +1,16 @@
+sudo: false
 language: go
+go:
+  - 1.6.2
```

Comment:
I think `tip` covers that.

Once 1.7 is released officially, you can change 1.6.2 to be 1.7 instead (and always keep tip as a fast-finish allow-failure target).